app/lib/features/pins/pages/pin_page.dart

```diff
@@ -30,39 +28,37 @@ class PinPage extends ConsumerWidget {
   ) {
     final spaceId = pin.roomIdStr();
     List<PopupMenuEntry> actions = [];
+    final pinEditNotifier = ref.watch(pinEditProvider(pin).notifier);
     final membership = ref.watch(roomMembershipProvider(spaceId));
     if (membership.valueOrNull != null) {
       final memb = membership.requireValue!;
       if (memb.canString('CanPostPin')) {
         actions.add(
           PopupMenuItem(
-            onTap: () => context.pushNamed(
-              Routes.editPin.name,
-              pathParameters: {'pinId': pin.eventIdStr()},
-            ),
+            onTap: () => pinEditNotifier.setEditMode(true),
             child: const Row(
               children: <Widget>[
-                Icon(Atlas.pencil_edit_thin),
+                Icon(
+                  Atlas.pencil_box_thin,
+                ),
                 SizedBox(width: 10),
                 Text('Edit Pin'),
               ],
             ),
           ),
         );
       }
-
       if (memb.canString('CanRedactOwn') &&
           memb.userId().toString() == pin.sender().toString()) {
         final roomId = pin.roomIdStr();
-        actions.addAll([
+        actions.add(
           PopupMenuItem(
             onTap: () => showAdaptiveDialog(
               context: context,
               builder: (context) => RedactContentWidget(
                 title: 'Remove this post',
```

Comment:
```suggestion
                title: 'Remove this pin',
```

just noticed...

Reply:
This has been fixed in [deebdb4](https://github.com/acterglobal/a3/pull/1366/commits/deebdb49c86fe50f9119df2060e87aaa0da9e9c5) :)